### DISCUSSION TITLE
Comando telecomandi Clima Daikin

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2499] [GENERALE - Discussioni generiche]
Ciao a tutti, 
vorrei integrare i telecomandi del clima del mio impianto di climatizazione.
gli split sono Daikin e ovviamente hanno i telecomandi a IR.
cosa consigliate?

ciao.

**Comment by snakuzzo** [comment]
Broadlink

**Reply by Matty1184** [reply]
modello consigliato? l'uso è solo per clima daikin

**Comment by indomus** [comment]
> Ciao a tutti, vorrei integrare i telecomandi del clima del mio impianto di climatizazione. gli split sono Daikin e ovviamente hanno i telecomandi a IR. cosa consigliate?
> 
> ciao.

integrare a cosa

**Reply by Matty1184** [reply]
poter attivare o disattivare gli split tramite HA

**Reply by KC-inDomus** [reply]
https://indomus.it/progetti/domotizzare-un-condizionatore-tradizionale-con-broadlink-e-home-assistant-v2/

**Reply by Matty1184** [reply]
grazie mille!
info... 
il mio split è FWT03GATNMV1 e il telecomando "dovrebbe" essere il WRC-HPC.
il codice infrarosso non esiste vero? ho guardato nelle varie guide ma no ho trovato nulla in merito.

**Reply by snakuzzo** [reply]
Con il broadlink li puoi catturare

**Reply by indomus** [reply]
> grazie mille! info... il mio split è FWT03GATNMV1 e il telecomando "dovrebbe" essere il WRC-HPC. il codice infrarosso non esiste vero? ho guardato nelle varie guide ma no ho trovato nulla in merito.

li catturi come spiegato in guida

**Reply by Matty1184** [reply]
ho un problema con la configurazione del dispositivo.
riesco solo ad associarlo all'app di broadlink e non a e-control, una volta associato non riesco però a trovare il modo di asseganrgli l'IP Fisso.
non trovo la sezione corretta, posso "pingare" il dispositivo ma non vi accedo da pagina web.
come devo fare? 
premesso: ho già asseganto un IP al MAc adress nel router, ma non mi piace questa soluzione!
